deliver/lib/deliver/runner.rb

```diff
@@ -275,9 +275,14 @@ def transporter_for_selected_team(upload: false)
       api_token = Spaceship::ConnectAPI.token
       api_key = if options[:api_key].nil? && !api_token.nil?
                   # Load api key info if user set api_key_path, not api_key
-                  { key_id: api_token.key_id, issuer_id: api_token.issuer_id, key: api_token.key_raw, is_key_content_base64: api_token.is_key_content_base64 }
+                  { key_id: api_token.key_id, issuer_id: api_token.issuer_id, key: api_token.key_raw }
                 elsif !options[:api_key].nil?
-                  options[:api_key].transform_keys(&:to_sym)
+                  api_key = options[:api_key].transform_keys(&:to_sym).dup
+                  if api_key[:is_key_content_base64]
+                    # key is still base 64 style if api_key is loaded from option
+                    api_key[:key] = Base64.decode64(api_key[:key])
+                  end
```

Comment:
💄  Code Style nitpick:
```suggestion
                  # key is still base 64 style if api_key is loaded from option                    
                  api_key[:key] = Base64.decode64(api_key[:key]) if api_key[:is_key_content_base64]
```